code/datums/diseases/advance/symptoms/fever.dm

```diff
@@ -36,6 +36,6 @@ Bonus
 	return
 
 /datum/symptom/fever/proc/Heat(mob/living/M, datum/disease/advance/A)
-	var/get_heat = (sqrt(21+A.totalTransmittable()*2))+(sqrt(20+A.totalStageSpeed()*3))
+	var/get_heat = (sqrt(max(21,21+A.totalTransmittable()*2)))+(sqrt(max(21,20+A.totalStageSpeed()*3)))

```

Comment:
So the result is actually similar to the minimal value before the negative values are added. Honestly this whole line should be replaced with readable clamp..